Packs/Symantec_Advanced_Threat_Protection/Integrations/integration-Symantec_Advanced_Threat_Protection.yml

```diff
@@ -384,25 +396,29 @@ script:
         addTime(p, 'start_time', startTime);
         addTime(p, 'end_time', endTime);
         add(p, 'next', next);
-        
+
         if (limit) {
             p.limit = parseInt(limit);
         }
         var incidents = [];
         var res = doReq('POST', type, p);
         if (res.obj.result && res.obj.result.length !== 0) {
             if (type == 'incidents') {
-                incidents = incidents.concat(getIncidentsFromIncidentsResult(res.obj.result));
+                var result = getIncidentsFromIncidentsResult(res.obj.result, startTime);
+                currentFetch = result[0];
+                incidents = incidents.concat(result[1]);
             } else {
-                incidents = incidents.concat(getIncidentsFromEventsResult(res.obj.result));
+                var result = getIncidentsFromEventsResult(res.obj.result, startTime);
+                currentFetch = result[0];
+                incidents = incidents.concat(result[1]);
             }
         }
         next = 'next' in res.obj ? res.obj.next : null;
-        
+
         if (next) {
             setLastRun({ time: startTime, next: next, end_time: endTime });
         } else {
-            setLastRun({ time: endTime });
+            setLastRun({ time: currentFetch ? currentFetch : startTime});
```

Comment:
are you sure it should be `startTime`?

Reply:
if no incident was fetched then I don't want the last run to be updated to `now` but to remain the same (as we do in other integrations)